EIPS/eip-5792.md

```diff
@@ -122,38 +136,59 @@ Returns the status of a call batch that was sent via `wallet_sendCalls`. The ide
 type GetCallsParams = string;
 
 type GetCallsResult = {
-  status: 'PENDING' | 'CONFIRMED';
+  callsId: `0x${string}`;
+  chainId?: `0x${string}`;
+  batchStatus: string; // See "Status Codes"
   receipts?: {
     logs: {
       address: `0x${string}`;
       data: `0x${string}`;
       topics: `0x${string}`[];
     }[];
     status: `0x${string}`; // Hex 1 or 0 for success or failure, respectively
-    chainId: `0x${string}`;
     blockHash: `0x${string}`;
     blockNumber: `0x${string}`;
     gasUsed: `0x${string}`;
     transactionHash: `0x${string}`;
   }[];
+  capabilitiesResultsData?: Record<string, any> | undefined;
 };
 ```
 
+##### Status Codes for `batchStatus` field
+
+| Code | Description |
+|------|-------------|
+| 100  | Batch has been received by the wallet but has not completed execution onchain (pending) |
+| 200  | Batch has been included onchain without reverts, receipts array contains info of all calls (confirmed) |
+| 400  | Batch reverted completely and only changes related to gas charge may have been included onchain (chain rules failure) |
+| 500  | Batch has not been included onchain and wallet will not retry (offchain failure) |
+
+Status codes follow these categories:
+* 1xx: Pending states
+* 2xx: Confirmed states
+* 4xx: Chain rules failures
+* 5xx: Offchain failures
```

Comment:
This is a total nit, but is we are using the metaphor of HTTP status codes here, shouldn't it be the other way round?

4xx: HTTP Client Error ≈ Offchain Error
5xx: HTTP Server Error ≈ Onchain Error

I would see "offchain" similarly to a HTTP Client, and "onchain" similarly to a HTTP Server.